src/main/java/com/loofah/graph/api/repositories/SkillsRepository.java

```diff
@@ -0,0 +1,32 @@
+package com.loofah.graph.api.repositories;
+
+import com.loofah.graph.api.models.Skill;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class SkillsRepository {
+
+    private List<Skill> skills;
+
+    @Autowired
+    public SkillsRepository(List<Skill> setUpSkills) {
+        this.skills = setUpSkills;
+    }
+
+    public List<Skill> getSkills() {
+        return this.skills;
+    }
+
+    public Skill getSkill(String id) {
```

Comment:
We'll need another ticket to handle errors. eg. if an id is supplied but not found, here I'm just returning null. GraphQL has inbuilt error handling, in this case we still return a 200 with a big stack trace in the Errors object in the response, which we might want to abstract.